pom.xml

```diff
@@ -18,6 +18,10 @@
             <id>public.onebusaway.org</id>
             <url>http://nexus.onebusaway.org/content/groups/public/</url>
         </repository>
+        <repository>
```

Comment:
Ah - but I guess MySQL wouldn't be embedded, then, right?  You'd need to install/setup MySQL database independently of the gtfs-realtime-validator?

I'd prefer to stick to embedded database options to make this tool as easy as possible to deploy.  What do you think the best embedded option would be that also has official Hibernate dialect support?  Maybe HSQLDB?

Nice find on the performance measurement benchmarks, by the way.